c/src/ml-api-common.c

```diff
@@ -762,6 +763,25 @@ _ml_tensors_data_clone_no_alloc (const ml_tensors_data_s * data_src,
   return ML_ERROR_NONE;
 }
 
+/**
+ * @brief Gets the max tensor rank for the ml-api version
+ * @note nnstreamer supports max rank 8 since 2.3.0
```

Comment:
I'm not sure which version to make rank lower.
Is it correct  rank 8 is support since 2.3.0..?

Reply:
@myungjoo @jaeyun-jung
NNStreamer would be released as v2.4.0 in Tizen 7.5. Should this line be `2.4.0`?

Reply:
IMO 2.3.0 (nnstreamer v2.2.0 for Tizen 7.0 release, v2.3.x is unstable, prod version before tizen 7.5 release but now published from ppa..)

Reply:
If you want a visible distinguishment, you can say "since 2.3.1".
Then, update nnstreamer version .